README.md

```diff
@@ -1,3 +1,64 @@
 # ecomm
 
```

Comment:
Se sobrar um tempo, dê uma olhada no manual de marcações do Markdown - https://docs.github.com/pt/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax